### PR TITLE
CLDR-15291 Fix emoji annotation collision errors

### PR DESCRIPTION
**Pull request by AEApple**
CLDR-15291

Fix emoji annotation collision errors

- [ ] This PR completes the ticket.

### CODE REVIEW

**Comment by AEApple** [comment]
I haven't fixed the Saint Martin flag collision error yet. Should we just remove both?

https://emojipedia.org/flag-st-martin/
https://emojipedia.org/flag-sint-maarten/

File: common/annotations/ha.xml [hunk 1]
```diff
@@ -2218,11 +2218,11 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		<annotation cp="🪢">igiya | karkata | ƙulla | ƙulli | ruɗani</annotation>
 		<annotation cp="🪢" type="tts">ƙulli</annotation>
 		<annotation cp="👓">ido | kayan ido | sutura | tabarau</annotation>
-		<annotation cp="👓" type="tts">tabarau</annotation>
+		<annotation cp="👓" type="tts">gilashin ido/annotation>
```
Comment (macchiati) [inline common/annotations/ha.xml:2221]: This needs a </ instead of a <

This causes an error in reading ha.xml

File: common/annotations/kab.xml [hunk 2]
```diff
@@ -1299,8 +1295,6 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		<annotation cp="✒" type="tts" draft="unconfirmed">imru n tricet aberkan</annotation>
 		<annotation cp="🖋" draft="unconfirmed">imru n lmidad</annotation>
 		<annotation cp="🖋" type="tts" draft="unconfirmed">imru n lmidad</annotation>
```
Comment (macchiati) [inline common/annotations/kab.xml:1297]: > ...delete both the St. Martin & Sint Maarten annotations flag emojis in kab...
After their translation of St. Martin we could put (FR) and after Sint Maaren (NL). That at least gives them both a name, and the translators can fix next time.

**Review by macchiati — CHANGES_REQUESTED** [review]
Looks good, with a couple of items to fix.

File: common/annotations/ha.xml (hunk 1 above)
Comment (srl295) [reply common/annotations/ha.xml:2221]:
```suggestion
		<annotation cp="👓" type="tts">gilashin ido</annotation>
```

**Review by srl295 — CHANGES_REQUESTED** [review]

File: common/annotations/kab.xml (hunk 2 above)
Comment (srl295) [reply common/annotations/kab.xml:1297]: (^ agree, edited for clarity)

Comment (AEApple) [reply common/annotations/kab.xml:1297]: Removed the duplicate values for St. Martin in regions.

**Review by srl295 — APPROVED** [review]

**Comment by srl295** [comment]
@AEApple  why dont' you merge this and i'll see if i can fix some of the remaining

**Comment by AEApple** [comment]
Done.

**Pull request merged**